pwa/src/pages/object_entities/[id]/[entity].tsx

```diff
@@ -2,13 +2,16 @@ import * as React from "react"
 import Layout from "../../../components/common/layout";
 import ObjectEntityForm from "../../../components/object_entities/dataForm";
 
-const IndexPage = ({id, entityId}) => {
+const IndexPage = (props) => {
+  const entity_objectId: string = props.params.id === "new" ? null : props.params.id
+  const entityId: string = props.params.entity === "new" ? null : props.params.entity
+
   return (
-    <Layout title='Object entity' subtext="Edit your object entity here">
+    <Layout title='Object entity' subtext="Create or edit your properties">
```

Comment:
Maybe this `subtext` should be a bit more descriptive? 

e.g. "Create or edit your object entities"?